backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultUploadService.java

```diff
@@ -70,12 +72,23 @@ public TestResultUpload processResultCSV(InputStream csvStream)
               response.getWarnings(),
               response.getErrors());
 
-      _repo.save(result);
+      if (response.getOverallStatus() != ReportStreamStatus.ERROR) {
+        _repo.save(result);
```

Comment:
Good call 👍 this brings us in alignment with how ReportStream's uploader handles this case